### PR TITLE
removed duplicate tests for @babel/highlight getChalk method

### PR DESCRIPTION
**Pull request by Letladi**
Removed duplicate tests for the @babel/highlight getChalk method.

### CODE REVIEW

File: packages/babel-highlight/test/index.js [hunk 1]
```diff
@@ -93,23 +93,5 @@ describe("@babel/highlight", function() {
         });
       });
     });
-
-    describe("when colors are supported", function() {
```
Comment (nicolo-ribaudo) [inline packages/babel-highlight/test/index.js]: Maybe we should test `when colors are not supported` instead?

Comment (Letladi) [reply packages/babel-highlight/test/index.js]: Maybe so @nicolo-ribaudo, currently I'm just learning the code and going through it so I can't say what should be done at this point 😅

Comment (nicolo-ribaudo) [reply packages/babel-highlight/test/index.js]: Yes, I think that it was duplicated because one of them was meant to test when colors were not supported

Comment (Letladi) [reply packages/babel-highlight/test/index.js]: Alright. I will update the pull request with that change.

Comment (Letladi) [reply packages/babel-highlight/test/index.js]: I've updated the pull request 👍🏼

**Pull request merged**